tests/ImageSharp.Web.Tests/Helpers/HMACUtilitiesTests.cs

```diff
@@ -10,39 +10,39 @@ public class HMACUtilitiesTests
         private static readonly byte[] HMACSecretKey = new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31 };
 
         [Theory]
-        [InlineData("The quick brown fox jumped.")]
-        [InlineData("Over the lazy dog.")]
-        public void CanGenerate256BitHMAC(string value)
+        [InlineData("The quick brown fox jumped.", "8ac58e6e983b9cdea7771a2f68b7430c8923ce43f690e23a51192f427fd7ece9")]
+        [InlineData("Over the lazy dog.", "2ea9f37afac5ae93e27d835cc6061ad84b31064ae29f451c9136e88ad5686a1e")]
+        public void CanGenerate256BitHMAC(string value, string expected)
         {
-            string expected = HMACUtilities.ComputeHMACSHA256(value, HMACSecretKey);
```

Comment:
Because checking `Assert.Equal(expected, actual);` doesn't make sense if they're both generated exactly the same way...

The only valuable assertion is whether the length is correct (okay and that it doesn't throw exceptions), but if it would generate a static string of the correct length, this test will still succeed!